versions/1.0.0-Draft.md

```diff
@@ -223,7 +223,7 @@ Describes the general information of the pricing of the a given plan.
 | :------------- | :------------:| :------------|
 | cost           | `number`      | **Optional** Units of cost associated with this service. Defaults to `0` if unspecified. |
 | currency       | `string`      | **Optional** Currency used to express the cost. Supported currency values are expressed in [ISO 4217](https://en.wikipedia.org/wiki/ISO_4217) format. Samples: `USD`, `EUR`, or `BTC` for US dollar, euro, or bitcoin, respectively. Defaults to `USD` if unspecified. |
-| billing        | `string`      | **Optional** Period used for billing. Possible values are: `onepay`, an unique payment before start using the service; `daily` , billed every day; `weekly`, billed every week; `monthly`, billed every month; `quarterly`, billed every quarter; `yearly`, billed every year. If it is not specified, the default value assumed is `monthly`|
+| billing        | `string`      | **Optional** Billing frequency. Possible values are: `onepay`, an unique payment before start using the service; `daily` , billed every day; `weekly`, billed every week; `monthly`, billed every month; `quarterly`, billed every quarter; `yearly`, billed every year. If it is not specified, the default value assumed is `monthly`|
```

Comment:
- Keep Period: 1 month period is correct. Monthly means frequency.
- Remove `daily` `weekly` `monthly` `quarterly` from the description.

Reply:
In my understanding, we discussed that periods for limitations is different from the billing. In the first case of a period, it was better to change values to remove the "-ly" suffix (e.g. `second`) but in the second case , as it is clearly a frequency,  we should keep the "-ly" suffix (e.g. `yearly`) starting on `monthly`.